### PR TITLE
Added full discography download

### PR DESCRIPTION
**Pull request by ghost**
This fixes Issue #150. It downloads the full discography, gives a list of URLs it will download, and will download both albums and tracks from an artist's music page.

### CODE REVIEW

File: bandcamp_dl/__main__.py [hunk 1]
```diff
@@ -84,6 +84,9 @@ def main():
         urls = Bandcamp.generate_album_url(arguments['--artist'], arguments['--track'], "track")
     elif arguments['--artist']:
         urls = Bandcamp.get_full_discography(arguments['--artist'], "music")
+        print(f"{len(urls)} albums/tracks have been found:")
```
Comment (Evolution0) [inline bandcamp_dl/__main__.py]: Other than this print I don't really have an issue with the PR. I just don't see a reason to clutter the terminal with anything more than the existing progress bar especially considering its not really a value add to print the urls.

If you want to dump them in debug that is fine though.

Comment (ghost) [reply bandcamp_dl/__main__.py]: Alright, I decided to remove it. It's already committed.

**Comment by Epictyphlosion** [comment]
Awesome! Now how do I input the command arguments correctly?

**Comment by Evolution0** [comment]
> Awesome! Now how do I input the command arguments correctly?

Should just need to pass the discography page as the URL.

**Pull request merged**